aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -682,7 +696,7 @@ impl AptosVM {
         // Also, do not move this after we run failure epilogue below, because this will load
         // module, which alters abort info. We have a transaction at version 596888095 which
         // relies on this specific behavior...
-        let status = self.inject_abort_info_if_available(module_storage, status);
+        let status = self.inject_abort_info_if_available(module_storage, traversal_context, status);
```

Comment:
We need to update the comment above this line, but also be careful with adding traversal context - maybe if the invariant re:traversal context does not hold, we return a different type of consistent error? (since it seems we only use traversal context to avoid the metadata call in a case which shouldn't happen (that's why I refer as checking an invariant)

Reply:
Removed the comment above, removed the invariant check - I think having unmetered access all the time with consistent error message is simple and acceptable 👍